docker-compose.yml

```diff
@@ -83,6 +83,8 @@ services:
       VELA_SERVER_SECRET: 'zB7mrKDTZqNeNTD8z47yG4DHywspAh'
       WORKER_ADDR: 'http://worker:8080'
       WORKER_CHECK_IN: 2m
+      QUEUE_PUBLIC_KEY: 'DXsJkoTSkHlG26d75LyHJG+KQsXPr8VKPpmH/78zmko='
+      QUEUE_ADDR: 'redis://redis:6379'
```

Comment:
```suggestion
```
These lines can be removed once https://github.com/go-vela/worker/pull/635 is merged. The latest worker image was using a server commit that had a bug